__test__/containers/big-movie-tile/BigMovieTile.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import BigMovieTile from '../../../src/js/containers/big-movie-tile/BigMovieTile.jsx';
+import { render } from 'enzyme';
+
+describe('BigMovieTile', () => {
+  it('should rendered correctly', () => {
```

Comment:
You have a yours mocks already so maybe would be better to just import them instead of copy-paste?